src/components/destination/index.js

```diff
@@ -15,13 +15,18 @@ class DestinationFullPage extends Component {
     } = this.props;
     const latitude = articleContent.location ? Number(articleContent.location.lat) : 0;
     const longitude = articleContent.location ? Number(articleContent.location.lon) : 0;
+    const isDraggable = window.innerWidth > 750;
```

Comment:
Is there a chance that this will still match tablets? Do we possibly want to consider a more "touch" oriented way of detecting this such as https://ctrlq.org/code/19616-detect-touch-screen-javascript?